src/Sulu/Component/Localization/Manager/LocalizationManagerInterface.php

```diff
@@ -30,5 +30,8 @@ public function getLocalizations(): array;
      */
     public function getLocales(): array;
 
+    /**
+     * @deprecated since Use the constructor instead
+     */
```

Comment:
When this is deprecated the LocalizationManagerInterface and the LocalizationProviderInterface are the same (just different method names). So I would suggest deprecating the entire interface and using the LocalizationProviderInterface everywhere.